docs/content/manual/manual.yml

```diff
@@ -925,6 +925,24 @@ sections:
             input: '{"a": 1, "b": 2, "c": 3}'
             output: ['{"a": 2, "b": 3, "c": 4}']
 
+      - title: "`pick(stream)`"
```

Comment:
I think we want `s/stream/pathexp/` here and below, since the closure passed here is passed to `path/1`, so it's really a path expression.

Reply:
Don't forget to update the docs to say `pathexps`.